posthog/async_migrations/utils.py

```diff
@@ -25,14 +25,20 @@ def execute_op(op: AsyncMigrationOperation, uuid: str, rollback: bool = False):
 def execute_op_clickhouse(sql: str, query_id: str, timeout_seconds: int):
     from ee.clickhouse.client import sync_execute
 
-    sync_execute(f"/* {query_id} */ " + sql, settings={"max_execution_time": timeout_seconds})
+    try:
+        sync_execute(f"/* {query_id} */ " + sql, settings={"max_execution_time": timeout_seconds})
+    except Exception as e:
+        raise Exception(f"Failed to execute ClickHouse op: sql={sql}, query_id={query_id}, exception={str(e)}")
```

Comment:
won't this get messy? maybe we want some new lines?

Reply:
yes, new lines are nice - we need them in the UI too later